src/DocTest/Parser.elm

```diff
@@ -67,32 +191,32 @@ importRegex =
     Regex.regex "^\\s{4}(import\\s.*)"
 
 
+functionNameRegex : Regex
+functionNameRegex =
+    Regex.regex "(\\w+)\\s:"
+
+
 commentRegex : Regex
 commentRegex =
-    Regex.regex "{-[^]*?-}"
+    Regex.regex "({-[^]*?-})\n([^\n]*)\\s[:=]"
 
 
 assertionRegex : Regex
 assertionRegex =
     Regex.regex "^\\s{4}(.*)"
 
 
-continuationRegex : Regex
-continuationRegex =
-    Regex.regex "^\\s{4}\\-\\-\\-\\s(.*)"
-
-
 expectationRegex : Regex
 expectationRegex =
     Regex.regex "^\\s{4}\\-\\->\\s(.*)"
```

Comment:
+/- 0 - it would be neat, but I'm not sure if that's what we want right now (best to just trust a "final" result)